### PR TITLE
Service areas and drive times for Department level

### LINKED ISSUE #393: Intersect Response Travel Time with Station regions
Determine performance of on-demand intersection spatial analysis verses caching information

### LINKED ISSUE #395: Display Geometry of Response Time roll-up on Map
Add Map layer to menu option for risk level coverage based on Response Times

### PR DESCRIPTION
**Pull request by ssontag55**
Created a route to get all station assets for each department
closes #395 closes #393

### CODE REVIEW

File: firecares/tasks/update.py [hunk 1]
```diff
@@ -448,3 +453,124 @@ def expand(values, weights):
...
+@app.task(queue='servicearea')
+def create_parcel_department_hazard_level_rollup_all():
+    """
+    Task for updating the servicearea table rolling up parcel hazard categories with departement drive time data
+    """
+    for fd in FireDepartment.objects.all():
+        get_parcel_department_hazard_level_rollup(fd.id)
+
+
+def get_parcel_department_hazard_level_rollup(fd_id=100214):
```
Comment (meilinger) [inline firecares/tasks/update.py]: Probably shouldn't have a default `fd_id`

File: firecares/tasks/update.py [hunk 2]
```diff
@@ -448,3 +453,124 @@ def expand(values, weights):
...
+    drivetimegeom0 = arcgis2geojson(drivetimegeom[2]['geometry'])
+    drivetimegeom4 = arcgis2geojson(drivetimegeom[1]['geometry'])
+    drivetimegeom6 = arcgis2geojson(drivetimegeom[0]['geometry'])
+
+    cursor = connections['nfirs'].cursor()
+
+    QUERY_INTERSECT_FOR_PARCEL_DRIVETIME = """SELECT sum(case when l.risk_category = 'Low' THEN 1 ELSE 0 END) as low,
+        sum(CASE WHEN l.risk_category = 'Medium' THEN 1 ELSE 0 END) as medium,
+        sum(CASE WHEN l.risk_category = 'High' THEN 1 ELSE 0 END) high,
+        sum(CASE WHEN l.risk_category is null THEN 1 ELSE 0 END) as unknown
```
Comment (meilinger) [inline firecares/tasks/update.py]: I doubt this is the case, but you might want to check that there aren't any parcels that have a `risk_category = trim('')`, if _ALL_ are `null` then we're good.

File: firecares/tasks/weather_task.py [hunk 3]
```diff
@@ -9,3 +10,25 @@ def collect_weather_noaa_warnings():
     This harvest can take up to 10 minutes to iterate through all warnings
     """
     WeatherWarnings.load_warning_data()
+
+
+@app.task(queue='weather-task', rate_limit='24/h')
+def cleanup_dept_weather_noaa_warnings():
```
Comment (meilinger) [inline firecares/tasks/weather_task.py]: The commit that has this change can be killed from this branch.  Actually, be sure that any of the weather-related commits are not included in this PR.  If they need to be pulled-out and moved to a separate PR, let me know, but I did just merge in the latest weather-related cleanup item.

File: firecares/utils/arcgis2geojson.py [hunk 4]
```diff
@@ -0,0 +1,229 @@
+"""
```
Comment (meilinger) [inline firecares/utils/arcgis2geojson.py]: NICE!  In the past, my method for converting ArcGIS REST json to geojson was running an `ogr2ogr` task in the background and reading the results from the resulting file (blech), this is SO MUCH BETTER!

File: firecares/weather/migrations/0004_auto_20171021_2107.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```
Comment (meilinger) [inline firecares/weather/migrations/0004_auto_20171021_2107.py]: Another case of changes that need to be pulled out from this PR/branch.

File: firecares/weather/models.py [hunk 6]
```diff
@@ -36,7 +36,7 @@ class WeatherWarnings(models.Model):
     expiration = models.DateTimeField(null=True, blank=True)
```
Comment (meilinger) [inline firecares/weather/models.py]: Pull out, part of the weather task.

File: firecares/weather/tests/test_weather.py [hunk 7]
```diff
@@ -69,7 +69,6 @@ def test_warning_service(self, mock):
```
Comment (meilinger) [inline firecares/weather/tests/test_weather.py]: Pull out, part of the weather task.

File: firecares/tasks/weather_task.py [hunk 8]
```diff
@@ -1,5 +1,6 @@
+from django.utils import timezone
```
Comment (meilinger) [inline firecares/tasks/weather_task.py]: Pull out, part of the weather task.

File: firecares/settings/base.py [hunk 9]
```diff
@@ -322,6 +322,7 @@
     Queue('quality-control', routing_key='quality-control'),
     Queue('slack', routing_key='slack'),
     Queue('weather-task', routing_key='weather-task'),
+    Queue('servicearea', routing_key='servicearea'),
```
Comment (meilinger) [inline firecares/settings/base.py]: We'll need to have the `servicearea` queue in the VM creation for this, see https://github.com/FireCARES/firecares-ansible/issues/11

File: firecares/tasks/update.py (hunk 2 above)
Comment (ssontag55) [reply firecares/tasks/update.py]: select distinct risk_category, count(risk_category) from parcel_risk_category_local group by risk_category  

no value other than null, low, medium and high
